ci/code_checks.sh

```diff
@@ -378,4 +378,13 @@ if [[ -z "$CHECK" || "$CHECK" == "typing" ]]; then
 fi
 
 
-exit $RET
+### Checking for standardized documentation ###
+echo "Checking which files in pandas/doc/source have standardized documentation"
+for entry in $(find ../doc/source/ -type f)
+do
+	if grep -q Pandas "$entry" || grep -q *pandas* "$entry"  ; then
+		echo "$entry"
+	fi
+done
+
+exit $RET
```

Comment:
think you need a newline here

Reply:
Sorry I'm not too sure what you mean, I'm new to this. Do you want me to add a new line after done?

Reply:
> Do you want me to add a new line after done?

yes please

Reply:
I have added a new line after done as requested!

Reply:
@willie3838 think you added the new line in the wrong spot. Should come at the end of this line not preceding

Reply:
> @willie3838 think you added the new line in the wrong spot. Should come at the end of this line not preceding

I have added your solution, let me know if it still does not resolve the issue.

Reply:
There needs to be a newline at the end of this file